documentation/rtd_requirements.txt

```diff
@@ -1,4 +1,5 @@
 # NOTE: relative paths are expected to be relative to the repository root
+#git+https://github.com/PEtab-dev/libpetab-python@develop#egg=petab
```

Comment:
```suggestion
petab>=0.2.1
```

Reply:
The current requirement is handled via setup.cfg. Will remove.